lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_gcp_pubsub.conf

```diff
@@ -0,0 +1,122 @@
+emqx_ee_bridge_gcp_pubsub {
+    desc_config {
+        desc {
+            en: """Configuration for a GCP PubSub bridge."""
+            zh: """GCP PubSub 桥接配置"""
+        }
+        label {
+            en: "GCP PubSub Bridge Configuration"
+            zh: "GCP PubSub 桥接配置"
+        }
+    }
+
+    desc_type {
+        desc {
+            en: """The Bridge Type"""
+            zh: """桥接类型"""
+        }
+        label {
+            en: "Bridge Type"
+            zh: "桥接类型"
+        }
+    }
+    desc_name {
+        desc {
+            en: """Bridge name, used as a human-readable description of the bridge."""
+            zh: """桥接名字，可读描述"""
+        }
+        label {
+            en: "Bridge Name"
+            zh: "桥接名字"
+        }
+    }
+
+    connect_timeout {
+        desc {
+          en: "The timeout when connecting to the HTTP server."
+          zh: "连接HTTP服务器的超时时间。"
+        }
+        label: {
+              en: "Connect Timeout"
+              zh: "连接超时"
+            }
+    }
+
+    max_retries {
+        desc {
+          en: "Max retry times if an error occurs when sending a request."
+          zh: "请求出错时的最大重试次数。"
+        }
+        label: {
+              en: "Max Retries"
+              zh: "最大重试次数"
+            }
+    }
+
+    pool_size {
+        desc {
+          en: "The pool size."
+          zh: "连接池大小。"
+        }
+        label: {
+              en: "Pool Size"
+              zh: "连接池大小"
+            }
+    }
+
+    pipelining {
+        desc {
+          en: "A positive integer. Whether to send HTTP requests continuously, when set to 1, it means that after each HTTP request is sent, you need to wait for the server to return and then continue to send the next request."
+          zh: "正整数，设置最大可发送的异步 HTTP 请求数量。当设置为 1 时，表示每次发送完成 HTTP 请求后都需要等待服务器返回，再继续发送下一个请求。"
+        }
+        label: {
+              en: "HTTP Pipelineing"
+              zh: "HTTP 管道"
+            }
+    }
+
+    request_timeout {
+        desc {
+          en: "HTTP request timeout."
+          zh: "HTTP 请求超时。"
+        }
+        label: {
+              en: "Request Timeout"
+              zh: "HTTP 请求超时"
+            }
+    }
+
+    payload_template {
+        desc {
+          en: "The template for formatting the outgoing messages.  If undefined, will send all the available context in JSON format."
+          zh: "用于格式化外发信息的模板。 如果未定义，将以JSON格式发送所有可用的上下文。"
+        }
+        label: {
+              en: "Payload template"
+              zh: "有效载荷模板"
+            }
+    }
+
+    pubsub_topic {
+        desc {
+          en: "The GCP PubSub topic to publish messages to."
+          zh: "要发布消息的GCP PubSub主题。"
+        }
+        label: {
+              en: "GCP PubSub Topic"
+              zh: "GCP公共子课题"
+            }
+    }
+
+    service_account_json {
+        desc {
+          en: "JSON containing the GCP Service Account credentials to be used with PubSub."
```

Comment:
we may need to add more details about what this JSON is, and how to obtain it etc.

Reply:
:heavy_check_mark: